Sources/Terminal/AngouriMath.Terminal.Lib/FSharpInteractive.fs

```diff
@@ -105,4 +92,6 @@ let createKernel () =
         |> (fun f -> Func<GenericChart.GenericChart, string> f)
         |> (fun f -> 
             Formatter.Register<GenericChart.GenericChart> (f, "text/plain"))
-        Result.Ok kernel)
+
+        return! ()
```

Comment:
Implement `Run` and you can avoid this https://fsharpforfunandprofit.com/posts/computation-expressions-builder-part3/